repository-data/development/src/main/resources/hcm-content/content/documents/lks/listing-pages.yaml

```diff
@@ -78,13 +78,13 @@
     jcr:mixinTypes: ['hippo:named', 'hippo:versionInfo', 'mix:referenceable']
```

Comment:
Could we reload `/content/documents/lks/listing-pages` ?

Reply:
Added.